main/exercise/question.class.php

```diff
@@ -123,22 +129,22 @@ public function getIsContent()
      * Reads question information from the data base.
      *
      * @param int   $id              - question ID
-     * @param array $course_info
+     * @param array $courseInfo
      * @param bool  $getExerciseList
      *
      * @return Question
      *
      * @author Olivier Brouckaert
      */
-    public static function read($id, $course_info = [], $getExerciseList = true)
+    public static function read($id, $courseInfo = [], $getExerciseList = true, $ptest = false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $TBL_EXERCISE_QUESTION is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)